binance-smart-chain-rpc/src/main.js

```diff
@@ -23,7 +23,7 @@ module.exports.run = async () => {
     ethereum,
   }
 
-  if (process.env.MIGRATE_ON_BOOTSTRAP === 'true') await require('@/share/bin/sequelizeMigrate')()
+  if (process.env.MIGRATE_ON_BOOTSTRAP === 'true') await require('@/share/sequelize/bin/sequelizeMigrate')()
```

Comment:
I see what you are saying. Since `postgres` is deployed once and connected to from the different services, your suggestion is definitely meaningful.

Do you think we could merge this for now since all `sequelize` related stuff are in the `share` folder? We could take a second shot at this later and remove the `sequelize` dependencies from all services other than `gateway`?